src/style.css

```diff
@@ -0,0 +1,57 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+}
+
+.mojo-logo-link {
+  height: 40px;
+  width: 40px;
+}
+
+.header {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  height: 60px;
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.menu-container {
+  display: flex;
+  list-style: none;
+}
+
+.menu-container > .menu-item + .menu-item {
```

Comment:
Instead this you can use pseudo classes `:not(:first-child)`.

Reply:
fixed